pkg/ignition/builder.go

```diff
@@ -129,6 +131,17 @@ func (b *ignitionBuilder) GenerateConfig() (config ignition_config_types_32.Conf
 		0644, false,
 		[]byte("[connection]\nipv6.dhcp-duid=ll\nipv6.dhcp-iaid=mac")))
 
+	if b.additionalNTPServers != "" {
+	    chronyConfig := []byte{}
+	    for _, server := range strings.Split(b.additionalNTPServers, ","){
+		    chronyConfig = append(chronyConfig, []byte(fmt.Sprintf("server %s iburst\n", server))[:]...)
```

Comment:
nit: it could potentially be less awkward to build chronyConfig as a strings.Builder and convert the resulting string to []byte in the very end.

Reply:
Done